tensorflow/compiler/xla/service/pattern_matcher.h

```diff
@@ -1940,6 +1993,17 @@ class HloInstructionPattern {
     return AppendImpl(HloInstructionPatternNameImpl(name));
   }
 
+  // Modifies the pattern to optionally match a unary operand with a given
+  // opcode before matching a given pattern.
+  template <typename OperandType, typename OperandImpl>
```

Comment:
Same here, these should not be called "Operand", and the variable shouldn't be called that.